### PR TITLE
Update docker command to docker compose v2

### PR DESCRIPTION
**Pull request by TharmiganK**
## Examples
N/A

## Checklist
- [ ] ~Linked to an issue~
- [ ] ~Updated the changelog~
- [ ] ~Added tests~
- [ ] ~Updated the spec~
- [ ] ~Checked native-image compatibility~

### CODE REVIEW

**Review by DimuthuMadushan — DISMISSED** [review]

File: ballerina/tests/resources/compose.yaml [hunk 1]
```diff
@@ -1,5 +1,3 @@
-version: '2'
```
Comment (ayeshLK) [inline ballerina/tests/resources/compose.yaml:1]: I think this version works with `docker compose` so we don't need to change it ?

File: examples/temperature-sensor/A sample temperature sensor using Ballerina MQTT.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ send an email to the user (Note that the email sending part is not implemented i
 1. [Install MQTT in your local machine](https://mosquitto.org/download/)
 2. [Use MQTT with docker](https://hub.docker.com/_/eclipse-mosquitto)
 
-* You can find the docker file used to set up the cluster for this example in [here](../../ballerina/tests/resources/docker-compose.yaml).
```
Comment (ayeshLK) [inline examples/temperature-sensor/A sample temperature sensor using Ballerina MQTT.md:23]: Do we hv to change the filename ?

File: ballerina/tests/resources/compose.yaml (hunk 1 above)
Comment (TharmiganK) [reply ballerina/tests/resources/compose.yaml:1]: Yes, it works with the warning as a backward compatibility
The recommendation is not to use a version

File: examples/temperature-sensor/A sample temperature sensor using Ballerina MQTT.md (hunk 2 above)
Comment (TharmiganK) [reply examples/temperature-sensor/A sample temperature sensor using Ballerina MQTT.md:23]: Not necessarily, from v2 the recommended name is compose.yaml, so I changed accordingly

**Review by ayeshLK — APPROVED** [review]

**Pull request merged**